### ISSUE TITLE
404 for AdminManageSitemap.aspx

### ISSUE THREAD

**Issue by andreasnylin** [Geta/SEO.Sitemaps#32]
Hello!

We experience an issue right now where Modules/Geta.SEO.Sitemaps/AdminManageSitemap.aspx results in a 404. Episerver.CMS 9.7.1 and latest version of Sitemaps.

Error occured	6/14/2016 3:42:39 PM
User IP	127.0.0.1
User Agent	Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.84 Safari/537.36
Url	http://urlremovedforprivacyreasons/Modules/Geta.SEO.Sitemaps/AdminManageSitemap.aspx
Referer	(none)
Exception details: HttpException: Not Found
Http status: 404 NotFound
Stack trace:

[HttpException: Not Found]

Have you experienced something similar?

We are having trouble understanding what causes the problem. I've debugged the code in the code behind file and it all seems to work OK. The problem seems to be related to SimplePage. I tried creating a new aspx-file in the modules-folder. The page works fine util i change it to inherit simple page. Then it also results in a 404.

**Comment by frederikvig** [comment]
Just tested with Episerver 9.10.2 and Sitemaps 1.5.7, without issues. Could you try upgrading and testing again? Closing this for now, please reopen if you still experience the same issue after updating to latest version.

**Comment by andreasdj** [comment]
Hello, we have the exact same issue, i.e. we get a 404 when navigating to Modules/Geta.SEO.Sitemaps/AdminManageSitemap.aspx using episerver 10.0.2.

I created a local copy and debugged it and we got into the code behind file and it was actually the base.OnInit(e) call that caused the 404. We have a special setup where we don't have a regular start page. Instead we have a site settings page (that lacks a view as we are only interested in the "all properties" view) that acts as a start page. It can in turn point out an arbitrary child page as the start page and we have created our own routehandling for the site root (I know it's a special case but we have our reasons).

An important note though, is that if I add a view for the site settings page it all works fine.
